Dockerfile

```diff
@@ -20,13 +21,14 @@ RUN conda install --name python2 --quiet --yes \
     joblib \
     markdown \
     pytables \
-    python-igraph
+    python-igraph 
 
 # RISE: "Live" Reveal.js Jupyter/IPython Slideshow Extension
 # https://github.com/damianavila/RISE
 RUN conda install --name python2 --quiet --yes -c bioconda zeroc-ice && \
     conda install --name python2 --quiet --yes -c damianavila82 rise && \
-    pip2 install py2cytoscape
+    conda install --name python2 --quiet --yes -c pdrops pygraphviz -y && \
+    /opt/conda/envs/python2/bin/pip install py2cytoscape pydot graphviz tqdm gseapy
```

Comment:
Minor point on code-style: when you have multiple packages can you put them on separate lines (as in the rest of this file) to make future diffs clearer?